timeseries/tests/unittests/models/ensemble/test_abstract.py

```diff
@@ -0,0 +1,71 @@
+from unittest import mock
+
+import pytest
+
+from autogluon.core.utils.exceptions import TimeLimitExceeded
+from autogluon.timeseries.models.ensemble import AbstractTimeSeriesEnsembleModel
+
+from ...common import DUMMY_TS_DATAFRAME, PREDICTIONS_FOR_DUMMY_TS_DATAFRAME, get_data_frame_with_item_index
+
+
+class DummyEnsembleModel(AbstractTimeSeriesEnsembleModel):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._model_names = []
+
+    def _fit(self, predictions_per_window, data_per_window, model_scores=None, time_limit=None, **kwargs):
+        self._model_names = list(predictions_per_window.keys())
+
+    def _predict(self, data, **kwargs):
+        return PREDICTIONS_FOR_DUMMY_TS_DATAFRAME
+
+    def remap_base_models(self, model_refit_map: dict[str, str]) -> None:
+        pass
+
+    @property
+    def model_names(self) -> list[str]:
+        return self._model_names
+
+
+class TestAbstractTimeSeriesEnsembleModel:
+    """Test the methods that are common to all ensemble models."""
+
+    @pytest.fixture()
+    def model(self):
+        yield DummyEnsembleModel()
+
+    @pytest.fixture()
+    def ensemble_data(self):
+        yield {
+            "predictions_per_window": {
+                "dummy_model": [PREDICTIONS_FOR_DUMMY_TS_DATAFRAME],
+                "dummy_model_2": [PREDICTIONS_FOR_DUMMY_TS_DATAFRAME],
+            },
+            "data_per_window": [DUMMY_TS_DATAFRAME],
+            "model_scores": {"dummy_model": 25.0, "dummy_model_2": 15.0},
```

Comment:
Nit: scores are always negative for time series, let's keep these negative too in case this becomes relevant in the future?
```suggestion
            "model_scores": {"dummy_model": -25.0, "dummy_model_2": -15.0},
```